src/misc/coloring.c

```diff
@@ -21,10 +21,14 @@
 #include "igraph_coloring.h"
 
 #include "igraph_interface.h"
+#include "igraph_memory.h"
 #include "igraph_adjlist.h"
 
 #include "core/indheap.h"
 #include "core/interruption.h"
+#include "core/genheap.h"
+
+/* Greedy coloring heuristics */
```

Comment:
```suggestion
/* COLORED_NEIGHBORS: Choose vertices based on the number of already coloured neighbours. */
```

I noticed that you keep referring to the heuristic based on the number of already coloured neighbour as "greedy" and contrasting it with DSatur. However, all methods in this file are various kinds of greedy colourings. DSatur is also a greedy colouring.